tests/test_spark_udfs.py

```diff
@@ -70,16 +70,95 @@ def test_udf_registration(spark):
     linker.predict()
 
 
-def test_udf_functionality(spark):
+def test_jaro(spark):
     data = ["dave", "david", "", "dave"]
     df = pd.DataFrame(data, columns=["test_names"])
     df["id"] = df.index
-    df_spark = spark.createDataFrame(df)
+    df_spark_jaro = spark.createDataFrame(df)
 
     linker = SparkLinker(
-        df_spark,
+        df_spark_jaro,
         settings,
-        input_table_aliases="test_df",
+        input_table_aliases="test_jaro_df",
+    )
+
+    sql = """
+        select
+
+        /* Output test names for easier review */
+        l.test_names as test_names_l, r.test_names as test_names_r,
+
+        /* Calculate jaro results for our test cases */
+        jaro_sim(l.test_names, r.test_names) as jaro_test
+        from test_jaro_df as l
+
+        inner join
+        test_jaro_df as r
+
+        where l.id < r.id
+    """
+
+    udf_out = linker.query_sql(sql)
+    # Set accuracy level
+    decimals = 4
+
+    # Test jaro-winkler outputs are correct
+    jaro_w_out = tuple(udf_out.jaro_test.round(decimals=decimals))
+    jaro_expected = (0.7833, 0.0, 1.0, 0.0, 0.7833, 0.0)
+
+    assert jaro_w_out == jaro_expected
+
+    # ensure that newest jar is calculating similarity . jw of strings below is 0.9440
+    assert spark.sql("""SELECT jaro_sim("MARHTA", "MARTHA")  """).first()[0] > 0.9
```

Comment:
Just use single quotes for these instead of `"""`?
```suggestion
    assert spark.sql('SELECT jaro_sim("MARHTA", "MARTHA")').first()[0] > 0.9
```